source/access/how_to_request_more_quota.rst

```diff
@@ -11,9 +11,32 @@ option depends on data storage policies of the site managing your VSC account,
 VO or Tier-1 project as well as on current capacity of the storage system.
 
 Before requesting more storage, please check carefully the
-:ref:`current data usage of your VSC account <quota>` and identify which
+:ref:`current data usage of your VSC account <checking disk usage>` and identify which
 file system needs a larger quota.
 
+.. _more quota inode:
+
+Requesting more inode quota
+===========================
+
+If you need more :ref:`inode quota <quota>`, you should first try to
+reduce the number of files as much as possible:
+
+- Clean up the files you no longer need.
+- Pack the remainder in a tarball, e.g. ``tar zcf myfiles.tgz file1 <...>
+  fileN``. If you need help on ``tar``, please consult the man page, *i.e.*, do:
+  ``man tar``.
+- If tarballs are not a suitable option for you, consider agregating your data
```

Comment:
so there is code that does a lot of small file readnig/writing, the user cannot change it to to wrap the file open in a tarball open en then a lookup to the file in the tarball, but somehow they can change to these suggestions? sounds a bit odd.

feel free to state that they should start with changing the code to use netdcdf or hdf5, since it's a much better format; and then worst case use tarballs (and worser case csv).
sqlite is also an option, but mariadb and postgres don't support direct db file access as far as i know.

Reply:
again, these are just suggestions.
i can imagine that tarballs are not a practical solution if the data need to be accessible for further analysis/processing
i did put the tarball option first because that is the simplest  (they don't have to change the code, just do an extra step afterwards). but no problem to change the order if you prefer.

for mariadb and postgres, i wasn't sure, i'll remove them